### ISSUE TITLE
Problems when pasting Youtube links in the bold editor

### ISSUE THREAD

**Issue by csk** [standardnotes/bold-editor#15]
The bold editor has an issue when pasting youtube videos.

I just wanted to paste a youtube link with a timestamp in it but the timestamp got removed by the parser. Also, when you paste a video it gets embedded but you can not click on it. If you do, nothing happens.

A PoC:

![Youtube link in bold editor issues](https://user-images.githubusercontent.com/153092/74187343-1c47e580-4c2b-11ea-9e75-1162ae9e4870.gif)

Please improve the bold editor :-)

**Comment by mobitar** [comment]
Thanks, have transferred this so we can better track it 👍

(Would be hilarious if this was a fake bug just to rickroll me 😂)

**Comment by csk** [comment]
I was not trying to rickroll you at all ...

fyi: i uploaded more details to my nice listed.io blog: [https://listed.to/@csk/11173/bold-editor-issues-found-so-far](https://www.youtube.com/watch?v=oHg5SJYRHA0) if you are interested.

**Comment by bryvin** [comment]
You got me...

**Comment by ningsongshen** [comment]
Yeah the videos not being clickable is something on their end I believe. I tried pasting the links provided into the [default demo](https://imperavi.com/redactor/) and the [video plugin demo](https://imperavi.com/redactor/plugins/video/) and both had unclickable videos.
